src/scss/attendee/effects/atom.scss

```diff
@@ -88,3 +88,10 @@ $-icon-size: 1.3rem;
     border-left: 1px solid theme.$sparkle--border;
   }
 }
+
+@mixin line-clamp($number-of-lines: 5) {
+  display: -webkit-box;
```

Comment:
is this a workaround only for `Webkit`?

Reply:
the persentage of browser support of it is pretty high
<img width="271" alt="image" src="https://user-images.githubusercontent.com/63313732/162736337-dc0b33d8-428b-4d86-aca0-0b42625d49cf.png">

https://caniuse.com/?search=line%20clamp